### PR TITLE
Add option to run other project tests

### PR DESCRIPTION
**Pull request by raptorxcz**
more at issue https://github.com/yonaskolb/XcodeGen/issues/461.

### CODE REVIEW

File: Sources/XcodeGenKit/SchemeGenerator.swift [hunk 1]
```diff
@@ -73,9 +73,48 @@ public class SchemeGenerator {
         return xcschemes
     }
 
+    private func getProject(for container: String?) -> PBXProj? {
```
Comment (yonaskolb) [inline Sources/XcodeGenKit/SchemeGenerator.swift]: Is this function used?

Comment (raptorxcz) [reply Sources/XcodeGenKit/SchemeGenerator.swift]: Yes you are right. Thanks. I removed that function.

**Comment by raptorxcz** [comment]
Hi @yonaskolb, I added tests.

**Comment by raptorxcz** [comment]
@yonaskolb Hi, could you check it?

**Comment by yonaskolb** [comment]
Hi @raptorxcz. I definitely appreciate the PR and what this is trying to fix. As mentioned in the linked issue I'd like to solve this problem in a generic way with proper multi-project support including some project referencing syntactic sugar.
If you require this specific fix for tests I would recommend using your fork until then.

**Comment by raptorxcz** [comment]
Ok, thanks for info. I am looking forward to workspace support. Until then I will use this feature. Is anywhere issue about workspace support? I did not find any related issue or merge request.

**Pull request closed without merging**